Regression/WarpX-tests.ini

```diff
@@ -64,7 +64,7 @@ branch = 18d0a2861d31c52c65752a1d5856f54e08699ce3
 [source]
 dir = /home/regtester/AMReX_RegTesting/warpx
 branch = development
-cmakeSetupOpts = -DAMReX_ASSERTIONS=ON -DAMReX_TESTING=ON -DWarpX_LIB=ON
+cmakeSetupOpts = -DAMReX_ASSERTIONS=ON -DAMReX_TESTING=ON -DWarpX_LIB=ON -DopenPMD_USE_ADIOS1=OFF
```

Comment:
Why is this needed? This will affect all of the CI and should probably be in a separate PR.

Reply:
Oops - this is just something I have to do on my local computer and I forget to remove this before submitting